lisa/base_tools/yum_config_manager.py

```diff
@@ -0,0 +1,45 @@
+from lisa.executable import Tool
+from lisa.operating_system import Posix
```

Comment:
This may cause cycle loop. You can add `# type: ignore` on `posix_os.install_packages(package_name)`